runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/DockerJobBundleFactory.java

```diff
@@ -82,6 +90,10 @@
   private final LoadingCache<Environment, WrappedSdkHarnessClient> environmentCache;
 
   public static DockerJobBundleFactory create(JobInfo jobInfo) throws Exception {
```

Comment:
Shall we make this private as we have JobBundleFactoryFactory now.

Reply:
Yes, it will be part of https://issues.apache.org/jira/browse/BEAM-4819